### PR TITLE
[native] Update full functional object and face detection tflite.py

### PR DESCRIPTION
**Pull request by dreamfinder2019**
Python version of object detection and face detection using Tflite and nnstreamer
now works full functonional. Both code detects objects or face.
PyCairo was used as visualization tool.
For face detection, the code shows only the largest detected face in the video,
while the other faces are covered with black rectangles.
Right now, These two codes have not yet implemented mutex or thread-related functions.

Signed-off-by: JANG JONG HA <jangjongha.sw@gmail.com>

### CODE REVIEW

**Review by cocony12 — APPROVED** [review]
LGTM

**Comment by dreamfinder2019** [comment]
Actually, I forgot to write this. The model of face_detection.py is based on detect_face.tflite in here.
(src: http://nnsuite.mooo.com/warehouse/nnmodels/) 
Should I make this model available for download via get_model.sh?

**Comment by yeseul4072** [comment]
That's a good idea.

**Review by soo-ni — APPROVED** [review]
LGTM 
In addition, I think downloading via get_model.sh is better than downloading from the link.

File: native/example_face_detection_tflite/nnstreamer_example_face_detection_tflite.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 """
 @file		nnstreamer_example_face_detection_tflite.py
 @date		4 Oct 2020
+<<<<<<< Updated upstream
```
Comment (gichan-jang) [inline native/example_face_detection_tflite/nnstreamer_example_face_detection_tflite.py:6]: The updated streams and stashed changes are not organized. Please Check them again.

**Comment by gichan-jang** [comment]
> Actually, I forgot to write this. The model of face_detection.py is based on detect_face.tflite in here.
> (src: http://nnsuite.mooo.com/warehouse/nnmodels/)
> Should I make this model available for download via get_model.sh?

I agree with @soo-ni's opinion. Please update the script.

**Review by myungjoo — COMMENTED** [review]
Please resolve merge conflicted contents.

**Comment by dreamfinder2019** [comment]
> > Actually, I forgot to write this. The model of face_detection.py is based on detect_face.tflite in here.
> > (src: http://nnsuite.mooo.com/warehouse/nnmodels/)
> > Should I make this model available for download via get_model.sh?
> 
> I agree with @soo-ni's opinion. Please update the script.
> And let's add .py files in meson.build and install them in `examples_install_dir`.

I didn't even think about meson.build. I'll work right away.

**Comment by dreamfinder2019** [comment]
In response to numerous fix requests, we have made the following changes:

0. Most importantly, the updated stream and stached changes were reorganized.
1. You can download the model for face detection using get_model.sh. Face detection models are available from the following sites. (http://nnsuite.mooo.com/warehouse/nnmodels/)
2. As the model download url of face_detection_tflite.py was confirmed, the name of the model read from the code was changed accordingly.
3. In README.md, the model download method is updated. But description is still in works.
4. By registering related files in meson.builds, python files will be also installed using build.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**